app/assets/javascripts/eventscripts/create_event.js

```diff
@@ -67,22 +67,82 @@ $( document ).on('turbolinks:load', function() {
         displayTypeField();
     });
 
+    function disableField(field){
+        $(field).find("select").attr("disabled", "disabled");
+        $(field).hide();
+    }
+
+    function enableField(field){
+        $(field).find("select").removeAttr("disabled");
+        $(field).show();
+    }
+
+    function enableDatesAndPlayerTypes(){
+        var enddatefield = $("#event_enddate_div");
+        var startdatefield = $("#event_startdate_div");
+        var durationfield = $("#event_duration_div");
+        var deadlinefield = $("#event_deadline_div");
+        var playertypes = $("#event_player_types");
+
+        enableField(enddatefield);
+        enableField(startdatefield);
+        enableField(durationfield);
+        enableField(deadlinefield);
+        enableField(playertypes);
+    }
+
+    function disableDatesAndPlayerTypes(){
+        var enddatefield = $("#event_enddate_div");
+        var startdatefield = $("#event_startdate_div");
+        var durationfield = $("#event_duration_div");
+        var deadlinefield = $("#event_deadline_div");
+        var playertypes = $("#event_player_types");
+
+        disableField(enddatefield);
+        disableField(startdatefield);
+        disableField(durationfield);
+        disableField(deadlinefield);
+        disableField(playertypes);
+    }
+
+    function disableMetricAndInitialValue() {
+        var metric = $("#event_metric_div");
+        var initialvalue = $("#event_initial_value_div");
+        disableField(metric);
+        disableField(initialvalue);
+    }
+
+    function enableMetricAndInitialValue() {
+        var metric = $("#event_metric_div");
+        var initialvalue = $("#event_initial_value_div");
+        enableField(metric);
+        enableField(initialvalue);
+    }
```

Comment:
Ooops, this is quite unfortunate, in the scope of #145 we decided to to quite the big refactor of the create progress, and now each event type has their own partial. So all the javascript isn't needed anymore. I talked about this with my PO @MarcelSchmidberger beforehand, so there must have been some misscomunication?

We'll have to figure out what the final call on this is.

Reply:
We got the same from our PO, however, for the refactoring of the create Event process, a whole other User story #227 exists. So I would suggest to do that refactoring in the context of that User Story.